modules/tinymce/src/themes/silver/main/ts/ui/dialog/IFrame.ts

```diff
@@ -51,7 +53,8 @@ const renderIFrame = (spec: IframeSpec, providersBackstage: UiFactoryBackstagePr
       uid: newSpec.uid,
       dom: {
         tag: 'iframe',
-        attributes
+        attributes,
+        classes: (isTransparent ? [ baseClass ] : [ baseClass, baseClass + '--opaque' ])
```

Comment:
Maybe better not sure:
```suggestion
        classes: (isTransparent ? [ baseClass ] : [ baseClass, `${baseClass}--opaque` ])
```